runtime/integration-tests/src/liquidity_pools/pallet/polkadot/setup.rs

```diff
@@ -0,0 +1,143 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
```

Comment:
Where did this file  (and kusama) version come from? Is this required now we use the `generic` module?